docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -85,3 +112,19 @@ A message indicates that this is a preconfigured connector.
 
 [role="screenshot"]
 image::images/pre-configured-connectors-view-screen.png[Pre-configured connector view details]
+
+If preconfigured connector belong to the preconfigured action type, it is disabled for the details preview.
+
+[role="screenshot"]
+image::images/pre-configured-action-type-managing.png[Connectors managing tab with pre-cofigured]
+
+
+[float]
+[[managing-pre-configured-action-types]]
+=== Managing preconfigured action types
+
+Clicking *Create connector* shows the list of available action types.
+Preconfigured action types are not included because you can't create a connector with a preconfigured action type.
```

Comment:
I believe the action types aren't included in the list if they are disabled via config, so I think this can be simplified:

> Disabled action types are not included.